### PR TITLE
add vertex log info

### PR DESCRIPTION
**Pull request by lucienicolier**

### CODE REVIEW

File: en/prepare/gnss.rst [hunk 1]
```diff
@@ -93,6 +93,21 @@ Examples:
...
+Vertex log
+..........
+
+The user assigns the role "digitizing logger" on a point layer.
+
+Go to QFieldSync --> project Properties
+
+.. container:: clearer text-left
+
+  .. figure:: ../images/vertex_log.png
```
Comment (m-kuhn) [inline en/prepare/gnss.rst:105]: Thanks, can you update the screenshot so it 

1. actually points to a layer
2. also includes a screenshot of the layer structure itself, ideally showing an example of a default value

@nirvn does have a sample project somewhere

Comment (lucienicolier) [reply en/prepare/gnss.rst]: @m-kuhn is it better?

**Review by m-kuhn — APPROVED** [review]
Beautiful :-)

File: en/prepare/gnss.rst [hunk 2]
```diff
@@ -93,6 +93,30 @@ Examples:
 I: Internal position source
 E: External (NMEA) position source
 
+Vertex log
+..........
+
+First usting GNSS variables in the point fields.
```
Comment (m-kuhn) [inline en/prepare/gnss.rst:99]:
```suggestion

It is possible to setup a log layer of the collected vertices.
This allows to keep track of meta data for each vertex like GNSS quality attributes and more.
To set this up, a point layer can be added to the project and attributes configured to store this information.
```

**Pull request merged**